### ISSUE TITLE
Training and Inference Setting Difference and Mode Collapse

### ISSUE THREAD

**Issue by JLiu-Edinburgh** [chenhu96/Self-Supervised-MRI-Reconstruction#2]
Hi, 

Thanks for the code and paper. It is interesting to train a model without the high-quality images.

The first concern about the method is the mode collapse which might happen to the "Difference Loss" in Fig.1. The loss aims at reducing the discrepancy in the predictions of the missing "high-frequency" elements. However, no further constraint is imposed to encourage the networks to learn such features and it might be less effective to prevent only learning the aliasing patterns, since the under-sampled image itself can minimize the training loss. Mode collapse is also a common issue in self-supervised frameworks, e.g. contrastive method and Siamese networks.

As mentioned in 2.4 Implementation Details, the input to the network in test is the original under-sampled raw data, e.g. 8✕, which is different from the training phase where a subset of points are used, e.g. ~16✕. Could you kindly explain how it is addressed? 

Moreover, if only a subset is used in inference, e.g. adopt an ensemble approach, the input quality can be manually degraded (also the case for training), which can severely increase the difficulties in producing high-fidelity reconstructions, even for models trained with fully sampled data. May I ask whether this case was considered in the proposed framework?

Look forward to any replies from you!
Best regards.
